packages/app/e2e-tests/HomePage.test.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { test, expect } from '@playwright/test';
+
+test('Should not throw `ResizeObserver loop completed with undelivered notifications`', async ({
+  page,
+}) => {
+  await page.goto('/');
+
+  const enterButton = page.getByRole('button', { name: 'Enter' });
+  await expect(enterButton).toBeVisible();
+  await enterButton.click();
+
+  await page.goto('/home');
+  await expect(
+    page
+      .frameLocator('#webpack-dev-server-client-overlay')
+      .getByText(
+        /ResizeObserver loop completed with undelivered notifications/,
+      ),
+  ).not.toBeVisible();
+});
+
+test('Should resize widgets vertically and horizontally', async ({ page }) => {
+  await page.goto('/');
+
+  const enterButton = page.getByRole('button', { name: 'Enter' });
+  await expect(enterButton).toBeVisible();
+  await enterButton.click();
+
+  await page.goto('/home');
+  await expect(page.getByText('Backstage Example App')).toBeVisible();
+
+  // Start editing mode
+  await page.getByRole('button', { name: /Edit/ }).click();
+  await expect(page.getByRole('button', { name: /Save/ })).toBeVisible();
+
+  // Resize the last installed widget
+  const widgetElement = await page.locator('.react-grid-item:nth-child(3)');
+  const defaultWidgetBox = { x: 1, y: 1, width: 1, height: 1 };
+  const widgetBoxBefore =
+    (await widgetElement.boundingBox()) ?? defaultWidgetBox;
+  const widgetResizeHandle = await widgetElement.locator(
+    '.react-resizable-handle',
```

Comment:
I wonder if we want to rely on class names. Just thinking thats something that can change without necessarily changing the element itself. But I also understand if you didn't have any alternatives here to happy to continue with this

Reply:
These are classes and elements added by the third-party library, let me see if we can customize them 🙂

Reply:
Done 🙌🏻 : https://github.com/backstage/backstage/pull/20784/commits/b6c3522b2a218c9af74cadcc548418984e521193

Reply:
sorry, didn't mean to add more work to this with my comment, I think what you did before then was maybe better actually, as it feels like this workaround introduced some changes and will require also release etc. Feel free to go with using classes that you had! i'll approve to unblock ✅

Reply:
Ah got it! No worries, I can drop the commit 🙂